app/views/docs/databases-relationships.phtml

```diff
@@ -0,0 +1,902 @@
+    <li>
+        <h3>Flutter</h3>
+        <div class="ide" data-lang="dart" data-lang-label="Flutter SDK">
+            <pre class="line-numbers"><code class="prism language-dart" data-prism>import 'package:appwrite/appwrite.dart';
+
+final client = Client()
+    .setEndpoint('https://[HOSTNAME_OR_IP]/v1')
+    .setProject('5df5acd0d48c2');
+
+final databases = Databases(client);
+
+await databases.updateDocument(
+  databaseId: 'marvel',
+  collectionId: 'movies',
+  documentId: 'spiderman',
+  data: {
+    'title': 'Spiderman',
+    'year': 2002,
+    'reviews': [
+        'review4',
+        'review5'
+    ]
+  },
+);</code></pre>
+        </div>
+    </li>
+    <li>
+        <h3>Apple</h3>
+        <div class="ide" data-lang="swift" data-lang-label="Apple SDK">
+            <pre class="line-numbers"><code class="prism language-swift" data-prism>import Appwrite
+
+let client = Client()
+    .setEndpoint("https://[HOSTNAME_OR_IP]/v1")
+    .setProject("5df5acd0d48c2")
+
+let databases = Database(client: client)
+
+databases.updateDocument(
+    databaseId: "marvel",
+    collectionId: "movies",
+    documentId: "spiderman",
+    data: [
+        "title": "Spiderman",
+        "year": 2002,
+        "reviews": [
+            "review4",
+            "review5"
+        ]
+    ]
+)</code></pre>
+        </div>
+    </li>
+    <li>
+        <h3>Android</h3>
+        <div class="ide" data-lang="kotlin" data-lang-label="Android SDK">
+            <pre class="line-numbers"><code class="prism language-kotlin" data-prism>import io.appwrite.Client
+import io.appwrite.services.Database
+
+val client = Client()
+    .setEndpoint("https://[HOSTNAME_OR_IP]/v1")
+    .setProject("5df5acd0d48c2")
+
+val databases = Database(client)
+
+databases.updateDocument(
+    databaseId = "marvel",
+    collectionId = "movies",
+    documentId = "spiderman",
+    data = mapOf(
+        "title" to "Spiderman",
+        "year" to 2002,
+        "reviews" to listOf(
+            "review4",
+            "review5"
+        )
+    )
+)</code></pre>
+        </div>
+    </li>
+</ul>
+
+<p>Relationships between documents can be deleted by <b>updating the relationship attribute</b> and removing the ID of the related document. If a relationship is two-way, documents in both of the related collections need to be updated. This approach <b>retains the documents</b> but deletes the link between them.</p>
+
+<p>If you need to delete the documents as well as the link between them, the approach depends on the <a href="/docs/databases-relations#on-delete">on-delete behavior</a> of a relationship.</p>
+
+<p>If the on-delete behavior is <b>restrict</b>, the link between the documents need to be deleted first before the documents can be deleted <b>individually</b>.</p>
+
+<p>If the on-delete behavior is <b>set null</b>, documents must be deleted <b>individually</b>. Before deleting a document, store the IDs in the relationship attribute so you don't lose track of related documents.</p>
```

Comment:
For set null, the documents don't need to be deleted individually, it means that any related documents will have their relationship set to null, effectively removing the relationship. 

E.g. for one-to-many movies to reviews, when deleting a movie with related reviews, the movie will be deleted, and the reviews will remain, but will have no linked movie.